libs/components/toast/src/lib/modules/toast/toast-instance.ts

```diff
@@ -10,16 +10,16 @@ export class SkyToastInstance {
    * An observable that indicates when the toast is closed.
    */
   public get closed(): Observable<void> {
-    return this._closed;
+    return this.#closed;
```

Comment:
Should we use the `asObservable` pattern for this property?
See: https://github.com/blackbaud/skyux/blob/43cfd830e1bad767110227142428a3dc5297a4e7/libs/components/core/src/lib/modules/overlay/overlay-instance.ts#L59